docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module

```diff
@@ -138,3 +138,13 @@ function hs_paragraph_types_field_widget_complete_paragraphs_browser_form_alter(
     $field_widget_form['widget'][$key] = $element;
   }
 }
+
+/**
+ * Implements hook_field_widget_complete_WIDGET_TYPE_form_alter().
+ */
+function hs_paragraph_types_field_widget_complete_options_select_form_alter(array &$field_widget_form, FormStateInterface &$form_state, array $context) {
+  // Change the default option label for field_accordion_style.
+  if (!empty($field_widget_form['widget']['#field_name']) && $field_widget_form['widget']['#field_name'] == 'field_accordion_style') {
```

Comment:
Formatting.

```suggestion
  if (
    !empty($field_widget_form['widget']['#field_name']) && 
    $field_widget_form['widget']['#field_name'] == 'field_accordion_style'
  ) { 
```

Reply:
If making this change, I would suggest to use `$context` instead of the `$field_widget_form`. I believe the field storage definition should be part of that array, so you can easily check the field name method.

Reply:
@pookmish @joegl Changed the condition to use `$context` instead. Didn't split it into multiple lines because the new condition is shorter.